### ISSUE TITLE
Security Issue with Pandoc via Chocolatey?

### ISSUE THREAD

**Issue by tajmone** [jgm/pandoc#3434]
I've updated Pandoc via Chocolatey to v1.19.2.1, but I've experienced some strange issues.

This is the Choco package:

https://chocolatey.org/packages/pandoc/1.19.2.1

When I query pandoc for it's version, I get v2.0 from the Chcoloatey install. I was expecting it to be 
v1.19.2.1. So I've installed over it the release from GitHub, and in fact it prints v1.19.2.1.

So, I'm not sure where the chocolatey package is actually downloading its version from... This could be a security breach and is worth looking into it. As a fact, after updating Pandoc via Chocolatey I'm having problems seeing pandoc on the system path, even though it's listed there -- even after reinstalling manually the GitHub release over it.

I'm afraid the Chocoloatey package might have compromised something on the system. If this is true (ie: if its not a problem on my system due to some previous malicious software being present) it might be worth issuing a warning about it.

Can anyone look into the issue? As I said, after this update there is no way I can see pandoc on my system path anymore...

**Comment by tajmone** [comment]
This is what I get when I query for Pandoc version:

```
C:\Users\XXXXXX\AppData\Local\Pandoc>pandoc -v
pandoc 2.0
Compiled with pandoc-types 1.17.0.4, texmath 0.9, skylighting 0.1.1.4
Default user data directory: C:\Users\P.K.Dick\AppData\Roaming\pandoc
Copyright (C) 2006-2016 John MacFarlane
Web:  http://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.
```

**Comment by jgm** [comment]
This was due to a packaging error on my part -- so, you needn't worry about a security breach.
I have replaced the defective package on our releases page, and Chocolatey should use the new msi.

**Comment by jgm** [comment]
PS. If you could notify chocolatey, I'd be grateful.  I barely know what chocolatey is.

**Comment by tajmone** [comment]
Thank you! I will notify Chocolatey.

By the way: the reason Pandoc wasn't seen on my system path is due to the fact that the Chocolatey package creates a `pandoc.exe` "shim" file in a Chocolatey folder which is on system path:

```
C:\ProgramData\chocolatey\bin\pandoc.exe
```

So when I try to invoke pandoc from CMD that is the file that gets executed (it comes first in the PATH hierarchy). I had to manually remove the file, and I am not sure why Chocolatey creates it, nor if it interferes only with my system (due to some custom settings) or if it's a general problem.

Anyhow, this "shim" file is a problem.

**Comment by tajmone** [comment]
Done: I've notified Chocolatey admins about the package (yesterday I had reported the potential threat via the abuse form, today I've reported the false allarm). I also mentioned the issue with the shim file interfering with pandoc usage, so hopefully that might get fixed also.

I also considered the fact that the pandoc shim file might be actually used by Chocolatey GUI to render the packages info page (they are in HTML!). But ChocoGUI is still working after removal of that shim file.

Next time I update pandoc via Chocolatey I will check if it recreates this shim file and prevents pandoc from being found on the system path.
